topics/cache.go

```diff
@@ -0,0 +1,127 @@
+package topics
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	sdk "github.com/rotationalio/go-ensign"
+)
+
+const DefaultTimeout = 15 * time.Second
+
+var (
+	// TODO: move to dedicated errors package
```

Comment:
Do we have a backlog issue for an errors package?

Reply:
We don't … not really sure the best way to go about it yet though; but I'll create a story for it.